src/modules/wallet/web3-react/hooks/useEagerlyConnect.ts

```diff
@@ -34,7 +36,10 @@ export function useEagerlyConnect() {
       connect(injectedWidgetConnection.connector)
     }
 
-    connect(gnosisSafeConnection.connector)
+    if (window.top !== window.self) {
+      connect(gnosisSafeConnection.connector)
+    }
```

Comment:
Oh, fancy!
I looked it up to understand what was happening, but the question is: why did it work before?
Could you add a comment as to why this is like that?

Reply:
Added comment.
Before we just did one excessive connection try even when app is not in Iframe